DTOs/ColourDTO.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Abp.Application.Services.Dto;
+using System;
+
+namespace RentCars
+{
+        public enum ColourDTO : EntityDto
+    {
+            black, white, red, yellow, green
+        }

```

Comment:
The proper casing for enum values is `PascalCase`.

Side note: I think there's no need to make a DTO for this enum. You can just call it `Colour` without inheriting from `EntityDto`